libres/lib/enkf/enkf_obs.cpp

```diff
@@ -1140,23 +1142,22 @@ stringlist_type *enkf_obs_alloc_matching_keylist(const enkf_obs_type *enkf_obs,
         return obs_keys;
 
     stringlist_type *matching_keys = stringlist_alloc_new();
```

Comment:
These changes looks good! But could I also ask for this to be a std::vector?

Reply:
I'll savor that for Monday I think.

Reply:
Leaving at is, or perhaps writing an issue is also a perfectly valid option